### PR TITLE
 feat: Switch to Yarn from npm

### PR DESCRIPTION
**Pull request by lipis**
Like most of the projects in this org

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -231,7 +231,8 @@
     "@types/resolve": "0.0.8",
     "cross-env": "^5.1.6",
     "mocha": "^5.2.0",
-    "vscode": "^1.1.21"
+    "vscode": "^1.1.21",
+    "yarn": "1.12.3"
```
[the remark this answers is not in the document]
Comment (lipis) [reply package.json]: So if somebody will try to run npm it will install it as dev dependency

**Comment by lipis** [comment]
Not sure why the tests are failing now, though.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (CiGit) [reply package.json]: But won't prevent anyone from using npm nor would use yarn instead ?

[the remark this answers is not in the document]
Comment (lipis) [reply package.json]: nope.. it should be smooth with npm as well.

**Comment by ntotten** [comment]
Merged through another commit.

**Pull request closed without merging**